src/test/scala/nl.knaw.dans.easy.deposit/servlets/IntegrationSpec.scala

```diff
@@ -101,7 +101,7 @@ class IntegrationSpec extends TestSupportFixture with ServletFixture with Scalat
     val uuid = createDeposit
 
     // upload files in a folder (more variations in UploadSpec)
-    val dataFilesBase = DepositDir(testDir / "drafts", "foo", UUID.fromString(uuid)).getDataFiles.get.bag.data
+    val dataFilesBase = DepositDir(testDir / "drafts", "foo", UUID.fromString(uuid)).getDataFiles.get.bag.data / DepositServlet.uploadRoot
```

Comment:
In this test you use the `uploadRoot` constant, whereas in other places you use `"original"` as a `String`. Please make sure that is consistent.